src/RwaFlipper.sol

```diff
@@ -16,22 +19,36 @@ contract RwaFlipper {
     }
 
     VatLike public vat;
```

Comment:
Should be immutable to save gas.

Reply:
This is currently using 0.5.12, so `immutable` isn't available here.

Reply:
Ah yes good point. Maybe it's worthwhile to bump up the version?

Reply:
`immutable` isn't available on this version of the compiler yet.